.evergreen/.evg.yml

```diff
@@ -201,7 +201,7 @@ functions:
       params:
         script: |
           ${PREPARE_SHELL}
-          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
+          MONGODB_VERSION=${VERSION} TOPOLOGY=${TOPOLOGY} AUTH=${AUTH} SSL=${SSL} STORAGE_ENGINE=${STORAGE_ENGINE} AWS=${AWS} sh ${DRIVERS_TOOLS}/.evergreen/run-orchestration.sh
```

Comment:
AWS="aws" lets run-orchestration use auth-aws.json to setup a mongod with the auth mech MONGODB-AWS